### PR TITLE
Added UWP Target and UWP Dpi

### PR DESCRIPTION
**Pull request by inforithmics**
#### Reference Issue
https://github.com/vvvv/SVG/issues/736

Svg.Skia issue
https://github.com/wieslawsoltes/Svg.Skia/issues/16

#### What does this implement/fix? Explain your changes.
Fixes Crash in UWP on Startup

used https://www.nuget.org/packages/MSBuild.Sdk.Extras/
for targeting UWP

### CODE REVIEW

**Comment by mrbean-bremen** [comment]
@wieslawsoltes - can you please have a look?

File: Source/SvgDocument.cs [hunk 1]
```diff
@@ -42,8 +42,11 @@ public override SvgOverflow Overflow
 
         private static int GetSystemDpi()
         {
+#if WINDOWS_UWP
+            var di = Windows.Graphics.Display.DisplayInformation.GetForCurrentView();
```
Comment (inforithmics) [inline Source/SvgDocument.cs:46]: This needs to run on UI Thread or else it crashes.

**Comment by inforithmics** [comment]
Before this can be merged I have to make it UI Thread save.

**Comment by inforithmics** [comment]
So I made it Thread Save and it should work now.

**Comment by FoggyFinder** [comment]
any updates?

**Comment by inforithmics** [comment]
Another possibility to achieve this would be to Reference Xamarin.Essentials and get the information from there.

I think the Main question is which way you want to go supporting with this library.

a) Implement platform support from externally. (like svg.skia for rendering on different platforms, then platform specific things like dpi need to be set externally Then this pull request  https://github.com/vvvv/SVG/pull/739 is needed).

b) Implement platform support internally than this library needs to be compiled for every platform.
UWP, Android, iOs, Linux, Mac (I could make other pull requests to support it).

c) Implementing a and b than both pull requests are needed

**Comment by inforithmics** [comment]
Another way fixing this would be to detect if i'm on UWP and return the Default 96 DPI

Code for detecting uwp platform

https://github.com/dotnet/corefx/blob/aa731cb68cd60128cc4b1efcd8b1a8e859750755/src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.cs#L23

Probably it would do this with a third pull request. What's your opnion about that?

**Comment by mrbean-bremen** [comment]
@H1Gdev , @wieslawsoltes : can you comment on this, please? Is this ready to be merged?

**Comment by wieslawsoltes** [comment]
I prefer solution from https://github.com/vvvv/SVG/pull/739 PR.

**Comment by mrbean-bremen** [comment]
Closing as discussed.

**Pull request closed without merging**